src/controllers/commands/CommandController.cpp

```diff
@@ -643,7 +643,85 @@ void CommandController::initialize(Settings &, Paths &paths)
             getApp()->windows->getMainWindow().getNotebook().getSelectedPage());
 
         currentPage->getSelectedSplit()->getChannelView().clearMessages();
+        return "";
+    });
 
+    this->registerCommand("/settitle", [](const QStringList &words,
+                                          ChannelPtr channel) {
+        if (words.size() < 2)
+        {
+            channel->addMessage(
+                makeSystemMessage("Usage: /settitle STREAM TITLE."));
+            return "";
+        }
+        if (auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get()))
+        {
+            auto status = twitchChannel->accessStreamStatus();
+            getHelix()->updateChannel(
+                twitchChannel->roomId(), "", "", words.mid(1).join(" "),
+                [channel](NetworkResult dank) {
+                    channel->addMessage(makeSystemMessage(
+                        QString("Updated title.").arg(dank.status())));
+                },
+                [channel] {
+                    channel->addMessage(
```

Comment:
Is it possible to forward the errors here so we can be a bit more intelligently informing the user if they're missing the scope or if the network connectivity is shit or what not? not a required change, just spitballing

Reply:
This would require changing Helix error handling which is out of scope of this PR.